src/autoComplete/index.tsx

```diff
@@ -168,19 +192,24 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
   onInputChange = ({ value }: { value: query }) => {
     this.currentValue = value
     if (this.hasMinCharsForSearch()) {
-      this.setState({ noResults: false, query: value }, this.searchForItems)
+      this.setState({
+        textfieldValue: String(value),
+        noResults: false, query: value },
```

Comment:
I'm surprised that prettier didn't change this indentation

Reply:
+1

Reply:
try to run `npm run prettier`, just in case?

Reply:
npm run prettier did help.
Is it called automatically? when?